packages/app-layout/src/AppLayoutContent.tsx

```diff
@@ -0,0 +1,112 @@
+import { Fragment, PropsWithChildren } from 'react';
+import { Prose } from '@ag.ds-next/react/prose';
+import { PageContent } from '@ag.ds-next/react/content';
+import { Callout } from '@ag.ds-next/react/callout';
+import { CoreProvider } from '@ag.ds-next/react/core';
+import { Text } from '@ag.ds-next/react/text';
+import { TextLink } from '@ag.ds-next/react/text-link';
+import { DirectionLink } from '@ag.ds-next/react/direction-link';
+import { hrefs } from './utils';
+import { Stack } from '@ag.ds-next/react/stack';
+
+export type ExpectedClaims = {
+	name?: string;
+	given_name?: string;
+	family_name?: string;
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	[others: string]: any;
+};
+
+const isEmptyString = (t: string | undefined) => (t?.trim() ?? '') === '';
+
+// This exists as its own package in ag.common.
+// Likely cleaner to re-use it here but makes consuming it more error prone (peer deps).
+const HelpCallout = () => {
+	return (
+		<CoreProvider>
+			<Callout title="Need more help?">
+				<Text>
+					Email{' '}
+					<TextLink href="mailto:exportservice@aff.gov.au">
+						exportservice@aff.gov.au
+					</TextLink>
+				</Text>
+				<Text>
+					Call <TextLink href="tel:1800571125">1800&nbsp;571&nbsp;125</TextLink>
+					, Monday to Friday, 9 am to 5 pm AEST
+				</Text>
+			</Callout>
+		</CoreProvider>
+	);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const MissingName = (props: PropsWithChildren) => (
+	<PageContent>
+		<Stack gap={3}>
+			<CoreProvider>
+				<DirectionLink href={hrefs.dashboard} direction="left">
+					Back to Dashboard
+				</DirectionLink>
+			</CoreProvider>
+
+			<Prose>
+				<h1>Single names in the Export Service</h1>
+				<p>
+					If you are legally known by a single name, you can use it to access an
+					Export Service account. Other services we offer may not accept a
+					single name and you will need to use a given and family name.
+				</p>
+				<p>Contact support if you need more help.</p>
+			</Prose>
+
+			<HelpCallout />
```

Comment:
You removed the Divider? I thought it was a standard part of our design pattern
<img width="519" alt="image" src="https://github.com/user-attachments/assets/6c1ced4d-0a3b-49e7-a65f-f63542c6c44d">